### PR TITLE
:sparkles: Add featured blog posts component

### PR DESCRIPTION
**Pull request by luxonauta**
desktop

mb

### CODE REVIEW

**Review by zenorocha — CHANGES_REQUESTED** [review]
Use 2 column layout

File: components/FeaturedBlogPosts.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    } else if (color === 'orange') {
+      return (
+        <LinesEllipsisLoose
+          text={title}
+          maxLine="2"
+          lineHeight="26"
+          className={`${styles.title} ${styles.titleOrange}`}
+        />
+      )
+    } else if (color === 'pink') {
```
[the remark this answers is not in the document]
Comment (luxonauta) [reply components/FeaturedBlogPosts.js]: I made these adjustments! ✨

**Comment by luxonauta** [comment]
> Use 2 column layout

The adjustments have already been made, Zeno! 🚀

**Review by zenorocha — CHANGES_REQUESTED** [review]
Make sure the centered container has the same width

**Pull request merged**